tests/device/test_iostream/test_iostream.ino

```diff
@@ -0,0 +1,33 @@
+#include <ESP8266WiFi.h>
+#include <BSTest.h>
+#include <sstream>
+#include <iostream>
+
+BS_ENV_DECLARE();
+
+void setup()
+{
+    Serial.begin(115200);
+    BS_RUN(Serial);
+}
+
+TEST_CASE("can print to std::cout", "[iostream]")
+{
+    std::stringstream test_stream("");
+    test_stream << "hello stream";
+
+	Serial.readString();
```

Comment:
Why is this needed ?

Reply:
Same comment as @d-a-v

Reply:
Was meant to clean up UART RX buffer just in case there was something in there. Chances are, the test will also work without this.

And I have messed up the indentation in this line...